GitDiffMargin.Extension/source.extension.vsixmanifest

```diff
@@ -18,12 +18,6 @@
         <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Community">
             <ProductArchitecture>amd64</ProductArchitecture>
         </InstallationTarget>
-        <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Community">
-            <ProductArchitecture>amd64</ProductArchitecture>
-        </InstallationTarget>
-        <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Pro">
-            <ProductArchitecture>amd64</ProductArchitecture>
-        </InstallationTarget>
```

Comment:
📝 This one is already matched by Community SKU

Reply:
I think that's the changes I did on master!

Reply:
Yes, this is just a simplification to master to remove some duplicate items that remain after the changes.

Reply:
Ok great, I will merge it for now so that it will come with the next release when I am back from vacations!